src/main/java/frc/trigon/robot/subsystems/elevator/simulationelevator/SimulationElevatorConstants.java

```diff
@@ -0,0 +1,61 @@
+package frc.trigon.robot.subsystems.elevator.simulationelevator;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.simulation.ElevatorSimulation;
+import frc.trigon.robot.simulation.MotorSimulationConfiguration;
+import frc.trigon.robot.subsystems.elevator.ElevatorConstants;
+
+public class SimulationElevatorConstants {
+    static final double VOLTAGE_COMPENSATION_SATURATION = 12;
+    static final boolean FOC_ENABLED = true;
+    private static final int NUMBER_OF_MOTORS = 2;
+    private static final double
+            GEAR_RATIO = 1,
+            MASS_KG = 10,
+            DRUM_RADIUS_METERS = 0.02,
+            MAXIMUM_HEIGHT_METERS = 1,
+            CONVERSIONS_FACTOR = 1,
+            P = 0,
+            I = 0,
+            D = 0,
+            KS = 0,
+            KV = 0,
+            KG = 0,
+            KA = 0,
+            MAXIMUM_ACCELERATION = 0,
+            MAXIMUM_VELOCITY = 0;
```

Comment:
all of these things are not the same, so you type the `private static final double` for whatever that is different than the others.